cmd/bucket-handlers-listobjects.go

```diff
@@ -107,9 +107,17 @@ func (api objectAPIHandlers) ListObjectsV2Handler(w http.ResponseWriter, r *http
 	// Validate the query params before beginning to serve the request.
 	// fetch-owner is not validated since it is a boolean
 	if s3Error := validateListObjectsArgs(prefix, marker, delimiter, maxKeys); s3Error != ErrNone {
+		// return empty response if invalid marker
+		if s3Error == ErrInvalidPrefixMarker {
```

Comment:
Add a TODO here to avoid this pattern when moving the errors constructively to a separate package. @poornas

Reply:
done.